app/main.py

```diff
@@ -1,25 +1,65 @@
-from abc import ABC
+class IntegerRange:
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
 
+    def __set_name__(
+            self, owner: "SlideLimitationValidator",
```

Comment:
Do not limit `owner` to just instances of `SlideLimitationValidator`, indicate that it could be any object in the type annotation to not limit the `IntegerRange` class. Apply this change to all related places